packages/web/src/components/track/GatedConditionsPill.tsx

```diff
@@ -44,12 +41,14 @@ export const GatedConditionsPill = ({
 
   return (
     <Button
-      className={cn(styles.button, className)}
+      className={className}
       size={buttonSize}
       onClick={onClick}
       color={isPurchase ? 'lightGreen' : 'blue'}
       isLoading={unlocking}
       iconLeft={showIcon ? IconLock : undefined}
+      // TODO: Add 'xs' button size in harmony
```

Comment:
@szonana thoughts? basically the buy button doesn't naturally fit with our harmony button sizes currently.